packages/origin-backend-client/src/DeviceClient.ts

```diff
@@ -30,16 +31,16 @@ export class DeviceClient implements IDeviceClient {
         return this.cleanDeviceData(data);
     }
 
-    public async getById(id: number): Promise<IDeviceWithRelationsIds> {
-        const url = `${this.endpoint}/${id}`;
+    public async getById(id: number, loadRelationIds = true): Promise<IDeviceWithRelationsIds> {
+        const url = `${this.endpoint}/${id}?loadRelationIds=${loadRelationIds}`;
         const { data } = await this.requestClient.get<void, IDeviceWithRelationsIds>(url);
 
         return this.cleanDeviceData(data);
     }
 
-    public async getAll(withMeterStats = false): Promise<IDeviceWithRelationsIds[]> {
+    public async getAll(withMeterStats = false, loadRelationIds = true): Promise<IDevice[]> {
```

Comment:
Any reason why `IDeviceWithRelationsIds` was replaced by `IDevice`?

Reply:
Because it doesn't always return these relation ids?

Reply:
Its because dependent on the loadRelationIds either IDeviceWithRelations or IDeviceWithRelationsIds can be returned